fuse/plugins/micro_physics/input.py

```diff
@@ -440,6 +436,13 @@ def _load_root_file(self):
         interactions["y_pri"] = ak.broadcast_arrays(xyz_pri["y_pri"], interactions["x"])[0]
         interactions["z_pri"] = ak.broadcast_arrays(xyz_pri["z_pri"], interactions["x"])[0]
 
+        if self.cut_by_eventid:
+            first_evtid = ak.min(interactions["evtid"])
+            start += first_evtid
+            stop += first_evtid
```

Comment:
Hey Pavel, doesn't this part of the code affect our selection?
Let's assume we have a file with with g4_evtid == 5, and we want to process from g4_evtid == 10.

Then I set
```
entry_start == 10
cut_by_eventid == True
```
Wouldn't this line then shift it as `start = 10 + 5 = 15` ?